docs/resources/dashboard.md

```diff
@@ -699,12 +698,45 @@ resource "datadog_dashboard" "free_dashboard" {
 - **template_variable** (Block List) The list of template variables for this dashboard. (see [below for nested schema](#nestedblock--template_variable))
 - **template_variable_preset** (Block List) The list of selectable template variable presets for this dashboard. (see [below for nested schema](#nestedblock--template_variable_preset))
 - **url** (String) The URL of the dashboard.
+- **widget** (Block List) The list of widgets to display on the dashboard. (see [below for nested schema](#nestedblock--widget))
 
 ### Read-Only
 
 - **dashboard_lists_removed** (Set of Number) A list of dashboard lists this dashboard should be removed from. Internal only.
 - **id** (String) The ID of this resource.
 
+<a id="nestedblock--template_variable"></a>
+### Nested Schema for `template_variable`
```

Comment:
```suggestion
### Nested schema for `template_variable`
```

H2 and below sentence-cased for docs

Reply:
Thanks for the review. These sections of the docs are auto-generated so unfortunately we will not be able to change these without making changes to the underlying library used to generate them.